invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/iiif_preview.html

```diff
@@ -9,11 +9,20 @@
 {%- extends config.PREVIEWER_ABSTRACT_TEMPLATE %}
 
 {%- block head %}
-{% for css_bundle in css_bundles %}
-{{ webpack[css_bundle] }}
-{% endfor %}
+<style>
+  .iiif-simple-preview {
+    display: flex;
+    justify-content: center;
+  }
+
+  .iiif-simple-preview img {
+    max-width: 100%
+  }
+</style>
```

Comment:
I think the styling inside HTML will fail on CSP. Could you move the CSS to site.overrides ?